backend/api/server.js

```diff
@@ -110,7 +110,7 @@ server.post(Path.CREATE_ACCOUNT, async (req, res, next) => {
   try {
     if (
       !fullNameValid(newUser?.fullName) ||
-      !emailValid(newUser?.email) ||
+      !emailValid(newUser?.email.trim()) ||
```

Comment:
Calling trim on `email` which may be undefined looks like it would cause an issue.
![image](https://github.com/user-attachments/assets/826458fb-4a20-49b4-8318-070adcea3a73)